tests/event-processing-tests/src/test/java/uk/gov/di/txma/eventprocessor/bdd/LambdaToS3StepDefinitions.java

```diff
@@ -23,9 +23,9 @@
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.time.Instant;
-import java.util.List;
-import java.util.Objects;
-import java.util.Stack;
+import java.time.ZoneOffset;
+import java.time.ZonedDateTime;
+import java.util.*;
```

Comment:
Please don't use `*` when doing Java imports... Please import that specific bits of the package.